packages/field-base/src/field-mixin.js

```diff
@@ -203,41 +257,24 @@ export const FieldMixin = (superclass) =>
       this._helperId = newId;
     }
 
-    /** @private */
-    __onHelperSlotChange() {
-      const current = this._currentHelper;
-
-      // Check fot slotted element node that is not the one created by this mixin.
-      const customHelper = this.__helperSlot.assignedElements({ flatten: true }).find((node) => node !== current);
-
-      if (customHelper) {
-        this.__updateHelperId(customHelper);
-
-        if (current && current.isConnected) {
-          current.remove();
-        }
-
-        this._applyCustomHelper();
-
-        this._currentHelper = customHelper;
-      }
-    }
-
     /** @protected */
-    _applyCustomHelper() {
-      const helper = this._helperNode.textContent;
-      if (helper !== this.helperText) {
-        this.helperText = helper;
+    _helperTextChanged(helperText) {
+      let helper = this._helperNode;
+
+      const hasHelper = this.__isNotEmpty(helperText);
+      if (hasHelper) {
+        // Create helper lazily
+        if (!helper) {
+          helper = this.__attachDefaultHelper();
+        }
```

Comment:
**nit:** `hasHelper` and `!helper` may be a bit confusing when used together. It would probably make it more clear for the reader if the `hasHelperText` naming was used:

```suggestion
      if (hasHelperText && !helper) {
        // Create helper lazily
        helper = this.__attachDefaultHelper();
      }
```